src/main/java/frc/robot/DTXboxController.java

```diff
@@ -52,10 +71,13 @@ public int getRawDPadPress() {
         if (!isDPadPressed) {
             isDPadPressed = true;
             return getPOV();
-        } else {
+        } 
+
+        else {
             if (getPOV() == -1) {
                 isDPadPressed = false;
             }
+            
```

Comment:
has this file been through the java formatter? this seems like a lot of formatting change